src/everest/bin/everest_script.py

```diff
@@ -145,10 +145,22 @@ async def run_everest(options: argparse.Namespace) -> None:
             logger.error(f"Failed to save optimization config: {e}")
 
         logging_level = logging.DEBUG if options.debug else options.config.logging_level
-        await start_server(options.config, logging_level)
+
         print("Waiting for server ...")
-        wait_for_server(options.config.output_dir, timeout=600)  # 10 minutes
+        logger.debug("Submitting everserver")
+        try:
+            await asyncio.wait_for(
+                start_server(options.config, logging_level), timeout=1800
```

Comment:
I am assuming this only happens for the testing hopefully. 

Is it possible to just use a different queue when running the tests to avoid queue congestion if that is indeed the source of the longer start times?

Reply:
Its always possible that someone starts an everest run on a cluster that has no available slots thus getting put in the queue. We have no way of handling this currently other than wait and timeout after time=X (in this case 1800s).

Ideally the info what we are in a queue should be propagated to the user